infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesRuntimeContext.java

```diff
@@ -75,6 +79,7 @@ public KubernetesInternalRuntime getRuntime() throws InfrastructureException {
 
     // there is cached runtime, restore cached one
     KubernetesRuntimeState runtimeState = runtimeStateOpt.get();
+    LOG.debug("Restoring runtime {} in workspace {}", runtimeState.getRuntimeId(), workspaceId);
```

Comment:
I guess `runtimeState.getRuntimeId()` will be logged as `RuntimeIdentityImpl{workspaceId='q', envName=....}`
I think it doesn't look well. Maybe it would be better to log something like 
```
LOG.debug("Restoring runtime `{}:{}:{}`", runtimeId.getWorkspaceId(), runtimeId.getEnvName(), runtimeId.getOwnerId());
```